recipe/meta.yaml

```diff
@@ -1,5 +1,5 @@
 {% set name = "kfp-tekton" %}
-{% set version = "1.1.1" %}
+{% set version = "1.1.0" %}
```

Comment:
@akchinSTC -- if we are patching a previous version, do we have to then subsequently have re-update the latest version after merging this PR?

Reply:
@ckadner - I just spoke to @Tomcli and after reading your link, we should probably open a support branches for 1.0.x and 1.1.x ? Ill rebase my PRs on those two and if we merge the PRs in the order (1.1.0 then 1.1.1) we should keep things spiffy and neat